evap/evaluation/models.py

```diff
@@ -1642,7 +1639,12 @@ def send_to_user(self, user, subject_params, body_params, use_cc, additional_cc_
             cc=cc_addresses,
             bcc=[a[1] for a in settings.MANAGERS],
             headers={'Reply-To': settings.REPLY_TO_EMAIL})
-        mail.attach_alternative(html_body, "text/html")
+
+        if self.html_body:
+            rendered_html_email_body = self.render_string(self.html_body, body_params)
+            html_template_params = dict({'email_body': rendered_html_email_body, 'email_subject': subject}, **body_params)
+            wrapped_html_email_body = render_to_string('email_base.html', html_template_params)
+            mail.attach_alternative(wrapped_html_email_body, "text/html")
```

Comment:
> ```python3
> if self.html_body:
>     # notice mismatch here: we use the words "body" and "content". Maybe we can find a unified naming scheme?
>     # Keep in mind: In the context of html, "body" is pretty strictly defined term that we're misusing here.
>     rendered_content = self.render_string(self.html_body, body_params)
>     wrapper_template_params = dict({'content': rendered_content, 'subject': subject}, **body_params)
>     wrapped_content = render_to_string('email_base.html', wrapper_template_params)
>     mail.attach_alternative(wrapped_content, "text/html")
> ```

As @He3lixxx already mentioned, the naming here is pretty difficult/confusing. I tried it this way, but it still feels weird. So maybe you have some ideas?

Reply:
> ```python3
> if self.html_body:
>     # notice mismatch here: we use the words "body" and "content". Maybe we can find a unified naming scheme?
>     # Keep in mind: In the context of html, "body" is pretty strictly defined term that we're misusing here.
>     rendered_content = self.render_string(self.html_body, body_params)
>     wrapper_template_params = dict({'content': rendered_content, 'subject': subject}, **body_params)
>     wrapped_content = render_to_string('email_base.html', wrapper_template_params)
>     mail.attach_alternative(wrapped_content, "text/html")
> ```
We agreed on this suggestion.